### DISCUSSION TITLE
Is the timezone uses UTC by default for scheduled maintenance?

### DISCUSSION THREAD

**Discussion by ajhalili2006** [upptime/upptime#365] [Q&A] [answered]
I want to start creating scheduled maintenance for our production Kutt instance to do some migrations to another Postgres server. Should I set the `start` and `end` times in the UTC timezone, like in GitHub Actions?

**Comment by AnandChowdhary** [comment, marked as answer]
Yes. :)

**Reply by ajhalili2006** [reply]
Okay. I'm using an timestamp generator to help me configure scheduled maintenances, because y'know I hate Math, minus the `+00:00` path.
